Update hooks.md

srhinow opened the pull request:
ActivateAccountListener -> AccountListener

Da die Klasse AccountListener heißt sollte der Aufruf dann ja auch den gleichen Namen haben.

aschempp commented:
Please move the description to the other `link_` inserttags.

lionel-m commented:
The problem is that this insert tag is not a link or an URL. It returns only a page name...

aschempp commented:
but that's the same with `link_title`?

lionel-m commented:
I modified.

srhinow committed:
Update hooks.md

ActivateAccountListener -> AccountListener

Da die Klasse AccountListener heißt sollte der Aufruf dann ja auch den gleichen Namen haben.

fritzmg reviewed (changes requested):
Danke @srhinow :) Der Fehler ist aber, dass die Klasse falsch heißt, denn der Hook heißt ja `activateAccount`. Die Klasse sollte `ActivateAccountListener` heißen.

zonky2 commented:
@fritzmg sollte es ein Issue geben, dass die Bezeichnung falsch ist? ggf. Ändern und Mapping und Deprecated setzen?

fritzmg commented:
@zonky2 was genau meinst du? Es ist ja nur in der Dokumentation flasch.

@srhinow du hättest nicht unbedingt einen neuen Pull Request machen müssen. Du hättest auch in deinem Fork im `patch-1` branch die Änderung machen können, dann wäre es auch im Pull Request automatisch aktualisiert ;)

Closing in favor of #267

zonky2 commented:
@fritzmg o.k. - dann habe ich das falsch verstanden..

srhinow commented:
ich habe genau in dieser Datei (weiter unten) einen weiteren fehler entdeckt. mach ich da jetzt da der #267 gepatched ist einen neuen auf?

fritzmg commented:
Yep
